src/Map/src/Exception/UnsupportedSchemeException.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Exception;
+
+use Symfony\UX\Map\Bridge;
+use Symfony\UX\Map\Renderer\Dsn;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+class UnsupportedSchemeException extends InvalidArgumentException
+{
+    private const SCHEME_TO_PACKAGE_MAP = [
+        'google' => [
+            'class' => Bridge\Google\Renderer\GoogleRendererFactory::class,
+            'package' => 'symfony/ux-map-google',
+        ],
+        'leaflet' => [
+            'class' => Bridge\Leaflet\Renderer\LeafletRendererFactory::class,
+            'package' => 'symfony/ux-map-leaflet',
+        ],
+    ];
```

Comment:
Not sure this is the best place to store this list :|

Reply:
This is what Symfony Messenger/Notifier/Mailer do, but I agree

Reply:
Well i never saw thsis before.. learning everyday..

Still find it very weird but hey, not something that important. Maybe can we just add a comment before both "lists" reminding the existence of the other one ?

Reply:
I'm now using the list from UXMapBundle